core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -205,7 +231,14 @@ private[spark] class EventLoggingListener(
   override def onBlockUpdated(event: SparkListenerBlockUpdated): Unit = {}
 
   // No-op because logging every update would be overkill
-  override def onExecutorMetricsUpdate(event: SparkListenerExecutorMetricsUpdate): Unit = { }
+  override def onExecutorMetricsUpdate(event: SparkListenerExecutorMetricsUpdate): Unit = {
+    // In order to avoid the logged event consumes too much storage size, taskMetrics would not
+    // be logged into event log file currently
+    val lightEvent = SparkListenerExecutorMetricsUpdate(
```

Comment:
the comment above the method is inaccurate (this is no longer a no-op obviously).  Can you change it something like "Track executor metrics for logging on stage start and end".

I'd also update the inner comment to something like "We only track the executor metrics in each stage, so we drop the task metrics as they are quite verbose".  and maybe rename "lightEvent" to "eventWithoutTaskMetrics"?